rpcs3/rpcs3qt/find_dialog.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <QDialog>
+#include <QTextEdit>
+#include <QLineEdit>
+#include <QPushButton>
+
+class find_dialog : public QDialog
+{
+public:
+	find_dialog(QTextEdit* edit, QWidget *parent = Q_NULLPTR, Qt::WindowFlags f = Qt::WindowFlags());
+	~find_dialog();
+
+private:
+	QTextEdit* m_text_edit;
+	QLineEdit* m_find_bar;
+	QPushButton* m_find_next;
+	QPushButton* m_find_last;
+
+private Q_SLOTS:
+	void find_next();
+	void find_last();
+};
```

Comment:
Not sure if we care about that, but there's no newline at end of the file.

Reply:
now it should be